starters/solidjs-tanstack-tailwind/src/index.css

```diff
@@ -1,3 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 body {
```

Comment:
Do we till need this though?

Reply:
Yes, based on the doc

https://tailwindcss.com/docs/guides/solidjs

Reply:
oh, will remove that in the next ticket
I don't wanna disrupt the style yet.

Reply:
But, i think the body will stay, but the code will go.